app/kumactl/pkg/cmd/root_context.go

```diff
@@ -130,51 +133,69 @@ func (rc *RootContext) catalog() (catalog.Catalog, error) {
 }
 
 func (rc *RootContext) CurrentDataplaneTokenClient() (tokens.DataplaneTokenClient, error) {
+	// todo(jakubdyszkiewicz) check enable/disable by checking cp config
```

Comment:
What does it mean ? Is it different from `if !components.Apis.DataplaneToken.Enabled()` below ?

Reply:
Will be resolved in https://github.com/Kong/kuma/issues/486